dist/Storable/Storable.xs

```diff
@@ -16,19 +16,18 @@
 #include <perl.h>
 #include <XSUB.h>
 
-#ifndef PATCHLEVEL
-#include <patchlevel.h>		/* Perl's one, needed since 5.6 */
+#ifndef PERL_CORE
```

Comment:
AFAIK, `PERL_CORE` will never be defined for Storable

Reply:
What is the correct way to get XS code to compile differently for a core compile compared to a CPAN compile?

Reply:
@Leont are you saying that once compiled in CORE, `PERL_CORE` is not set?
if this is true, then yes I need to change the logic here

Reply:
going to fix it with something like `#ifndef PERL_VERSION_LT`

Reply:
fixed by 1a559f4c7f881104afc36a15a74d5e2dd368dd2d